sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/BuilderHelper.java

```diff
@@ -76,6 +82,61 @@ static void configureConnectionString(String connectionString, Consumer<String>
         credentialSetter.accept(new StorageSharedKeyCredential(accountName, accountKey));
     }
 
+    static void parseEndpoint(String endpoint, Consumer<String> endpointSetter, Consumer<String> accountNameSetter,
```

Comment:
I'd prefer to return an object with all these parts populated, similar to your bloburl parts. THis is like having multiple "out String foo" method parameters

Reply:
Done